integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/services/IntegrationBridgeService.java

```diff
@@ -45,8 +46,11 @@
 
   private String minVersion;
 
-  public IntegrationBridgeService(String minVersion) {
```

Comment:
Does not any application down use this constructor?

Reply:
This constructor must not be used anymore.